business_requirement/README.rst

```diff
@@ -0,0 +1,265 @@
+
+You can modify the default numbering sequence in Settings/Technical/Sequences&Identifier/Sequences.
+
+Search for Business Requirement sequence and alter it if necessary.
+
+Tags
+^^^^
+
+You can create and assign tags for your business requirements in Business Requirements/Configuration/Bus. Req. Category
+
+|image2|
+
+.. |image2| image:: business_requirement/static/img/bus_req_tags.png
+   :width: 600 px
+   :alt: Define Tags
+
+
+Master project
+^^^^^^^^^^^^^^
+
+You can define a master project linked to the business requirement.
+
+Followers from the project will automatically be added to the business requirements.
+
+
+Usage
+=====
+
+Simple BR
+^^^^^^^^^
+1. Define the header information
+
+* Master Project (Partner is the one related to the master project)
+* Priority (low, Medium, High)
+* Change request: Is it a change request? (currently only informational with n model or action)
+* Tags: any relevant tag for the business.
+* Owner and approver by default
+
+|image3|
+
+.. |image3| image:: business_requirement/static/img/bus_req_tags2.png
+   :width: 800 px
+   :alt: Input header information
+
+2. Input the customer story, scenario and gap (simple html editor with image and text)
+
+|image4|
+
+.. |image4| image:: business_requirement/static/img/bus_req_cust_story.png
+   :width: 600 px
+   :alt: Input customer story, scenario, gap
+
+3. Confirm the Business requirement (for BR User and Manager)
+   At that stage the Customer story/Scenario/Gap is not modifiable anymore
+
+|image5|
+
+.. |image5| image:: business_requirement/static/img/bus_req_confirmed.png
+   :width: 800 px
+   :alt: Confirm your business requirement
+
+4. Approve the Business requirement (for BR Manager)
+
+|image6|
+
+.. |image6| image:: business_requirement/static/img/bus_req_approved.png
+   :width: 800 px
+   :alt: Confirm your business requirement
+
+5. Once your requirement is finished and delivered you can set it as Done
+
+|image8|
+
+.. |image8| image:: business_requirement/static/img/bus_req_done.png
+   :width: 800 px
+   :alt: Confirm your business requirement
+
+6. Alternatively, you can cancel the BR (in case it is not relevant or mistake) or drop it (when customer makes the decision to discontinue it)
+
+|image9|
+
+.. |image9| image:: business_requirement/static/img/bus_req_cancel.png
+   :width: 800 px
+   :alt: Cancel your business requirement
+
+|image10|
+
+.. |image10| image:: business_requirement/static/img/bus_req_drop.png
+   :width: 800 px
+   :alt: Drop your business requirement
+
+
+Sub-business requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^
+User can create sub business requirements for a given BR for complex cases.
+
+This is a simple parent/child relation (see roadmap).
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/140/8.0
+
+Known issues / Roadmap
+======================
+
+* Multi-company management
```

Comment:
Partially: the field exist but there is no complete ACL and it has not been fully tested.
Effort to get there is small though